cpp/ps0000/ps0000soln.cpp.cpp

```diff
@@ -1,16 +1,9 @@
-/******************************************************************************
+#include <iostream> // 2/2
 
-Welcome to GDB Online.
-GDB online is an online compiler and debugger tool for C, C++, Python, Java, PHP, Ruby, Perl,
-C#, OCaml, VB, Swift, Pascal, Fortran, Haskell, Objective-C, Assembly, HTML, CSS, JS, SQLite, Prolog.
-Code, Compile, Run and Debug online from anywhere in world.
-
-*******************************************************************************/
-#include <iostream>
-
-int main()
+int main() // 1/1
 {
-    std::cout<<"Hello World!"<<"\n";
+    std::cout << "Hello, World!\n"; // 3/4
```

Comment:
- [x] used `<<` for outputting instead of `>>`
- [x] used `\n` or `std::endl` in the right way
- [x] avoided syntax error by typing `;` at the end of the line
- [ ] output is as required `Hello, World!` in a single line